keyboards/shorty/keyboard.json

```diff
@@ -0,0 +1,54 @@
+{
+    "manufacturer": "Hawtkeys",
+    "keyboard_name": "shorty",
+    "maintainer": "hawtkeys",
+    "bootloader": "rp2040",
+    "diode_direction": "ROW2COL",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP28", "pin_b": "GP27"},
+            {"pin_a": "GP3", "pin_b": "GP4"}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["GP14", "GP13", "GP12", null, null],
```

Comment:
The use of `null` here has no value without any corresponding code and use within the layout macro?